images/macos/provision/core/aws.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash -e -o pipefail
 
+source ~/utils/utils.sh
+
 echo Installing aws...
-curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+AWSCLIURL="https://awscli.amazonaws.com/AWSCLIV2.pkg"
+download_with_retries $AWSCLIURL "."
```

Comment:
Is it better to download to `/tmp` like you have done in other places?

Reply:
Done